packages/product-feed-persoo/src/Resources/translations/messages.cs.po

```diff
@@ -0,0 +1,15 @@
+msgid ""
+msgstr ""
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Language: cs\n"
+
+msgid "Persoo Shopping product feed"
```

Comment:
I do not see this key anywhere in the code base

Reply:
I have dumped all translations, thanks for mentioning